README.md

```diff
@@ -65,6 +65,15 @@ Rovněž spusťe SPARQL dotazy ze složky `al-db-server/lucene`.
 `docker-compose stop al-termit-server ; docker-compose --env-file=.env.local up -d al-termit-server`
 
 8. Ověř, že Výrobní linka běží. V případě lokálního nasazení je její URL `http://localhost/modelujeme`.
+
+## Obnova databáze slovníků z publikovaných dat
+
+Služba `al-db-server` představuje RDF úložište obsahující pracovní verze slovníků a uživatelé systému. Uživatelé systému lze kdykoli smazat a přegenerovat pomocí služby `al-auth-server`, která ukláda zdrojové data o uživatelích pomocí služby `al-auth-server-db`. Pracovní verze slovníků lze publikovat do [Sémantického slovníku pojmů](https://xn--slovnk-7va.gov.cz/) (SSP). 
+
+V případě, že jsou všechny slovníky publikované, je možné RDF úložistě přemazat a naimportovat slovníky znovu z SSP. Po obnovení úložište (viz. body týkající se `al-auth-serveru` v sekci [Nasazení výrobní linky](https://github.com/opendata-mvcr/sgov-assembly-line/edit/main/README.md#nasazen%C3%AD-v%C3%BDrobn%C3%AD-linky)) je potřeba obnovit i RDF reprezentaci uživatelů systému v tomto úložišti. Záznam uživatele se v RDF úložišti vytvoří automaticky při libovolné aktualizaci uživatele pomocí uživatelského rozhraní na adrese `/modelujeme/sluzby/auth-server/`. Po aktualizaci všech uživatelů je obnova databáze slovníku hotova. 
```

Comment:
```suggestion
V případě, že jsou všechny slovníky publikované, je možné RDF úložistě přemazat a naimportovat slovníky znovu z SSP. Po obnovení úložište (viz. body týkající se `al-auth-serveru` v sekci [Nasazení výrobní linky](https://github.com/opendata-mvcr/sgov-assembly-line/edit/main/README.md#nasazen%C3%AD-v%C3%BDrobn%C3%AD-linky)) je potřeba obnovit i RDF reprezentaci uživatelů systému a TermIt ontologie v tomto úložišti. TermIt ontologii je zapotřebí naimportovat ze souboru `termit-ontologie/termit-ontologie.trig` pomocí grafického rozhraní databáze na adrese `/modelujeme/sluzby/db-server/import`. Záznam uživatele se v RDF úložišti vytvoří automaticky při libovolné aktualizaci uživatele pomocí uživatelského rozhraní na adrese `/modelujeme/sluzby/auth-server/`. Po aktualizaci všech uživatelů je obnova databáze slovníku hotova. 
```

Reply:
I unresolved this conversation ... I remember this comment of @Mighantos to include note about TermIt ontology ... but I did it by saying:
"Po obnovení úložište (viz. body týkající se `al-auth-serveru` v sekci [Nasazení výrobní linky](https://github.com/opendata-mvcr/sgov-assembly-line/edit/main/README.md#nasazen%C3%AD-v%C3%BDrobn%C3%AD-linky))" 

Beside TermIt ontology I would have to add also note about SPARQL queries related to `al-db-server/lucene`, so instead of state-ing it again here I just referenced "Nasazení výrobní linky" section.


Thus my plan is to revert this change and I will think of how to make it more clear to you.

Reply:
bylo to otevřené více než týden a bez komentu, tak jsem to commitnul, přišlo mi to v pohodě. Pokud se tiu to nelíbí, předělej to jiným způsobem

Reply:
@blcham Ok I get it now. I would suggest specifying that it also talks about "5. V al-db-serveru..." of the list of steps in "Nasazení výrobní linky".

Reply:
good point, fixing al-auth-serveru --> al-db-serveru

Reply:
@Mighantos please check/approve pull request